includes/Core/Authentication/Owner_ID.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Class Google\Site_Kit\Core\Authentication\Owner_ID
+ *
+ * @package   Google\Site_Kit\Core\Authentication
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Authentication;
+
+use Google\Site_Kit\Core\Storage\Setting;
+
+/**
+ * Owner_ID class.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+class Owner_ID extends Setting {
+
+	/**
+	 * The option_name for this setting.
+	 */
+	const OPTION = 'googlesitekit_owner_id';
+
```

Comment:
Maybe we should keep overriding the `get` method in this class, simply calling `parent::get()` and casting to `int`. WordPress saves this as a string in the DB no matter whether we cast it to an integer before, so I think overriding makes sense to ensure `get` returns an integer.

Reply:
I think you're right. I thought the sanitize callback was enough but I confirmed that if the value is loaded from the DB that it will return a string. I'll update method and tests to cover this.

Reply:
Ok. it's updated.